### DISCUSSION TITLE
How to expose the mitmproxy instance to the public Internet and open the mitmweb control interface on the cloud server?

### DISCUSSION THREAD

**Discussion by freshworker** [mitmproxy/mitmproxy#6178] [Q&A]
1. Hello, I have asked this question before and you answered：
 You can modify the [web_host](https://docs.mitmproxy.org/stable/concepts-options/#web_host) option to change the IP address mitmproxy is listening on. However, be careful about exposing your mitmproxy instance to the public internet.
But I don't understand，How to expose my mitmproxy instance to the public internet.。

2. When I try to change the listening IP address using 
`mitmweb --web-host <cloud server IP>`
the error report shows:
![图片](https://github.com/mitmproxy/mitmproxy/assets/77287677/ad0a2a0f-501c-4815-8c56-f104773598ae)

**Comment by ThomasBucaioni** [comment]
Hello,
It seems you're trying to send all the ingoing traffic from the Internet through mitmproxy? Maybe there are other monitoring tools best suited for traffic analysis. Mitmproxy is rather for you to catch what is going out when you browse outside

**Reply by arnau-zerod** [reply]
what tools ?
